### ISSUE TITLE
Multiple Screens

### ISSUE THREAD

**Issue by distraktd** [taviso/hotcorner#11]
Is it possible to get this working on multiple screens? I can't see an option for this in the compile section...
-edit forgot to say, nice work!

**Comment by taviso** [comment]
Do you mean multiple monitors? I use two monitors and it "just works", but file an issue if it doesn't and I'll look into it.

**Comment by distraktd** [comment]
I have a slightly different situation. I have a laptop and a monitor connected to it. I run the executable as admin (although same issue if i run it normally). However I only get a "hotcorner" on my laptop screen and not on the connected monitor. I'm assuming your two monitors are connected to a desktop?

**Comment by instinxx** [comment]
Just a guess and further info:
I am running a desktop with 2 monitors. When running the precompiled 1.3, only my "main" display has the hotcorner. My monitors are of different sizes and different resolutions however so I am unsure if the app is detecting coordinates for the corner of my main which does not map to the other monitor?

I dont need it to give a hotcorner on both monitors however, this is fine for my needs. Thanks!

**Comment by distraktd** [comment]
Not a bad guess, my laptop display is a different res to the monitor so it could be that. A hot corner on my main monitor is great although would be nice to have it on both, not the end of the world though.

**Comment by thewavelength** [comment]
For me, it only works on the main monitor, having three monitors all equal in size.
Edit: Maybe a more hard-to-solve problem than I originally thought: https://social.msdn.microsoft.com/Forums/vstudio/en-US/34bd062d-2100-4925-bdfa-9f98b881e7dc/winapi-getcursorpos-returning-incorrect-pointers-with-multiple-monitors?forum=vcgeneral

**Comment by distraktd** [comment]
Yeah, my setup is now a desktop with 2 monitors of equal resolution and as before I only get the hotcorner on the primary display. Although it's still fine for what I need. 

Oh I hadn't noticed that,  I might take a look at it post exams if I get the chance and see if there's a solution for my setup. If I find one then I'll be sure to share it in the case it works for others :)

**Comment by Ortega-Dan** [comment]
@distraktd Just go ahead and compile and use pull request #12  ... that does what you are looking for in a great way... I do use it on Windows 10 and even on a Win Server 2016 we have for development.
